### ISSUE TITLE
Error from simple_random on dashboard

### ISSUE THREAD

**Issue by jdanielnd** [splitrb/split#272]
I'm getting this error message when I try to access "/split":

    RuntimeError - Alpha and beta parameters must be positive. Received a = 22 and b = -9.:

From:

    simple-random/simple_random.rb:96:in `beta'

**Comment by yagudaev** [comment]
Just hit the same problem. I use forcing a parameter with the URL override option. It initially worked fine and completed the experiment with the actual selection and not the override. But then I hit this after I restarted my server and went through the process again.

In other words, I finished the experiment twice I believe, one with the override parameter and another time after that (also with the override). I got 

```
Alpha and beta parameters must be positive. Received a = 4 and b = -1
```

**Comment by robquesada** [comment]
Same error here. Not resolved yet.

**Comment by igor-brishkoski** [comment]
Same thing.

**Comment by robquesada** [comment]
Just to let you know, my error was that in the code was not clear when the test started and when it finished. Sometimes it wasn't starting but finishes, so the error triggered.

What I did to avoid it was to change the name of the metrics, so the alfa and beta counters start again (a=0, b=0), and then I analyze the code to be sure that the start/finish of the a/b test was logically correct.

**Comment by igor-brishkoski** [comment]
Thanks @robquesada. That worked. So the issue is that I finish the experiment more times than I start the experiment?

**Comment by robquesada** [comment]
Yeah, that it seems to be.

**Comment by yagudaev** [comment]
Shouldn't the gem detect this? There is a user session associated with the finish action. A user should not be able to finish the same experiment more than once. Also if the experiment hasn't started, the user should not be able to finish it.

**Comment by theB264** [comment]
I had this same error.  Somehow in making a new feature my staging environment's data became invalid, causing the error at the top of this issue.  After much investigation, I ended up being able to fix it in the console, it was not a (current version) code problem.  I thought I'd share what fixed it.  The first three commands get information and the last one fixed the problem because completed_count was bigger than participant_count for one experiment. ```Split.redis.keys
Split.redis.hget("your_experiment:1", 'participant_count')
Split.redis.hget("your_experiment:1", 'completed_count')

Split.redis.hset("your_experiment:1", 'completed_count', 3)```

**Comment by derricklannaman** [comment]
@theB264 That worked for me. Thanks!

**Comment by divineforest** [comment]
the fix is in the master

**Comment by andrew** [comment]
Fixed by #408
